plugins/standard/applets/conference/conference-calls.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+if(isset($_GET['json'])):
+
+	$ci =& get_instance();
+
+	$service = new Services_Twilio($ci->twilio_sid, $ci->twilio_token);
+
+	$conferences = $service->account->conferences->getIterator(0, 50, array("Status" => "in-progress"));
+
+	$res = array();
+	
+	foreach($conferences as $call) {
+		$res[$call->friendly_name] = array(
+			'date_created' => date("F j, Y, g:i a",strtotime($call->date_created)),
+			'friendly_name' => $call->friendly_name,
+			'status' => $call->status,
+			'duration' => $call->duration
+		);		
+	}
+	
+	die(json_encode($res));
+	
+endif;
+
+?>
+
+<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.2.6/jquery.min.js"></script>
+<script>
+
+	function join_call(conference_name){
+		
+		window.parent.Client.call({ 
+			'conference_name' : conference_name,
+			'muted' : 'false',
+			'beep' : 'true',
+			'Digits' : 1});
+		
+		return false;
```

Comment:
Returning false to cancel a click action is dirty. This should be done through jQuery listeners. 

```javascript
<script type="text/javascript">

function joinConference(params) {
	params = $.extend(params, { 'Digits': 1 });
	window.parent.Client.call(params)
}

jQuery(function($) {
	$('#calls').on('click', '.join', function(e) {
		e.preventDefault();
	
		joinConference({
			'conference_name': $(this).data('friendly-name'),
			'muted': 'false',
			'beep': 'true'
		});
	});

	$('#calls').on('click', '.listen', function(e) {
		e.preventDefault();
	
		joinConference({
			'conference_name': $(this).data('friendly-name'),
			'muted': 'true',
			'beep': 'false'
		});
	});
});

</script>

<a class="join" data-conference-name="conf-friendly-name">Join</a>
<a class="listen" data-conference-name="conf-friendly-name">Listen</a>
```

The jQuery '.on()' method will register a listener on the `#calls` table that will respond to events that bubble from `.listen` and `.join` elements. This allows you to add/remove elements to the table and have them trigger registered events.